src/test/java/api/loans/CheckOutByBarcodeTests.java

```diff
@@ -1148,6 +1152,27 @@ public void cannotCheckOutWhenItemLimitIsReachedForBookMaterialTypeAndLoanTypeIs
     assertThat(secondBookTypeItem, hasItemStatus(AVAILABLE));
   }
 
+  @Test
+  public void canCheckOutWithdrawnItem() {
+    final IndividualResource withdrawnItem = itemsFixture
+      .basedUponSmallAngryPlanet(ItemBuilder::withdrawn);
+
+    assertThat(withdrawnItem.getJson(), withdrawn());
+
+    final IndividualResource response = loansFixture
+      .checkOutByBarcode(withdrawnItem, usersFixture.steve());
+
+    assertThat(response.getJson(), allOf(
+      hasOpenStatus(),
+      hasItem(),
+      hasJsonPath("action", "checkedout"),
+      hasJsonPath("itemId", withdrawnItem.getId().toString()),
+      hasJsonPath("item.id", withdrawnItem.getId().toString())
```

Comment:
Could these be part of the item checks rather than separate?

Reply:
I've removed this statement, because it is not valuable for the case.